src/Consolonia.Core/Infrastructure/ConsoleWindow.cs

```diff
@@ -153,14 +153,12 @@ public void Show(bool activate, bool isDialog)
 
         public void Hide()
         {
-            // toplevel never hides
-            throw new NotImplementedException();
+            // console window can't do this.
```

Comment:
Lets better call `ConsoloniaPlatform.RaiseNotSupported` and add ignore at https://github.com/jinek/Consolonia/blob/c13a8b28846b9da01823b66087d0e84644397bd6/src/Consolonia.Core/Infrastructure/ConsoloniaPlatform.cs#L130
We can have one single enum value for all these cases.